### DISCUSSION TITLE
Glossary and acronym lists are missing

### DISCUSSION THREAD

**Discussion by mrcarvalho** [joaomlourenco/novathesis#400] [Q&A]
Good afternoon. 

The glossary list and the acronyms don't appear when I compile the file (I attach the error that appears). I can't figure out why. How can I solve the problem?

<img width="465" alt="Error" src="https://user-images.githubusercontent.com/121574211/209854273-27cb1581-5631-4aa8-90ae-f236fe43c40e.png">

**Comment by joaomlourenco** [comment]
Please use "latexmk -pdf" to compile the document.  It should work.

**Reply by mrcarvalho** [reply]
Good morning. I am using overleaf to write the dissertation and compile the pdf. Is the solution the same ?

**Reply by joaomlourenco** [reply]
In overleaf it should just work.
You must have some error while defining your glossary and LaTeX (in Oevrleaf) quits on trying to make the glossary.
Just besides the "Compile" green button, don't you have a red square with a number inside??  Those are errors… click in the red square and start fixing the errors.  Only when the red square turns orange, LaTeX will be orking properly.

**Comment by RSantos94** [comment]
I'm also using Overleaf (with latest version from GitHub (6.10.18)) and have the same problem. No errors, just some warnings

Acronyms is showing, but glossary isn't. Both are active in 4_files.tex

On the raw log it's showing 

"pdfTeX warning (dest): name{glo:IMU} has been referenced but does not exist, replaced by a fixed one"

I have the entry on the glossary and I'm calling it with \gls{IMU} (for this instance, but happens with all other glossary entries)
